instance-scheduler/main_test.go

```diff
@@ -394,7 +802,7 @@ func TestStopStartTestInstancesInMemberAccount(t *testing.T) {
 										},
 									},
 								},
-								// instance-scheduling = skip-auto-start, therefore skip auto start, but not stop, acted upon: 1
+								// DBinstance-scheduling = skip-auto-start, therefore skip auto start, but not stop, acted upon: 1
```

Comment:
this comment is wrong, this is an EC2 instance test!

Reply:
this has been corrected